Quaver.Shared/Skinning/SkinKeys.cs

```diff
@@ -548,11 +548,11 @@ private void LoadLaneSpecificElements()
 
                 // HitObjects
                 if (UseSameHitObjects)
```

Comment:
Just throwing this out there, but can we rename "UseSameHitObjects" to "UseHitObjectsAsHitHoldObjects" solely for descriptive purposes over on the wiki, and in the skin.ini file.